python/pyspark/sql/tests/connect/test_parity_dataframe.py

```diff
@@ -77,6 +77,11 @@ def test_to_pandas_from_mixed_dataframe(self):
     def test_toDF_with_string(self):
         super().test_toDF_with_string()
 
+    # TODO(SPARK-46261): DataFrame.withColumnsRenamed should respect the dict ordering
```

Comment:
May I ask why `TODO` JIRA has the same title with this PR?

Reply:
oh, it was a mistake , i changed it to SPARK-46261

Reply:
Could you update this comment too?
```
- # TODO(SPARK-46261): DataFrame.withColumnsRenamed should respect the dict ordering
+ # TODO(SPARK-46261): Python Client DataFrame.withColumnsRenamed should respect the dict ordering
```